src/maker/config.rs

```diff
@@ -14,32 +14,20 @@ pub struct MakerConfig {
     pub port: u16,
     /// RPC listening port
     pub rpc_port: u16,
-    /// Time interval between connection checks
-    pub heart_beat_interval_secs: u64,
-    /// Time interval to ping the RPC backend
-    pub rpc_ping_interval_secs: u64,
-    /// Time interval ping directory server
-    pub directory_servers_refresh_interval_secs: u64,
-    /// Time interval to close a connection if no response is received
-    pub idle_connection_timeout: u64,
     /// Absolute coinswap fee
     pub absolute_fee_sats: Amount,
     /// Fee rate per swap amount in ppb.
     pub amount_relative_fee_ppb: Amount,
     /// Fee rate for timelocked contract in ppb
     pub time_relative_fee_ppb: Amount,
```

Comment:
We need to explain what is PPB and how to calculate this value. Just a two lines of code comment is enough. 

I would suggest moving this to const for now and exposing it later in later releases. It's not a super important feature. In fact keeping it constant at 0, for initial public beta testing will make the swap fee calculations easier as well as user-facing configs simpler.

Later on we can always add it back as a "new cool feature". Better not to confuse the first users with it.